application/src/test/java/org/opentripplanner/model/plan/paging/cursor/PageCursorFactoryTest.java

```diff
@@ -181,30 +201,31 @@ private void assertPageCursor(
     Instant expLat,
     Duration expSearchWindow,
     PageType expPageType,
-    Boolean hasDedupeParams
+    Boolean hasDeduplicationParams,
+    Boolean hasGeneralizedCostMaxLimit
   ) {
     assertEquals(expEdt, pageCursor.earliestDepartureTime());
     assertEquals(expLat, pageCursor.latestArrivalTime());
     assertEquals(expSearchWindow, pageCursor.searchWindow());
     assertEquals(expPageType, pageCursor.type());
-    assertEquals(hasDedupeParams, pageCursor.itineraryPageCut() != null);
+    // This is related to the DeduplicationPageCut class.
```

Comment:
```suggestion
```
I do not understand this comment. The input parameter name could be renamed here. Or you could explain in JavaDoc on the method. Comments in the code should explain why - the intention when it can not be deduced from the code. If the code is unclear - refactor and make it clear instead. Do not write what the code does in plain english.

Reply:
There was some discussion related to this here: https://github.com/opentripplanner/OpenTripPlanner/pull/6474#discussion_r2019309984. I think `hasDedupeParams` is clear enough, so I changed it back and removed the comment.